decidim-core/app/helpers/decidim/layout_helper.rb

```diff
@@ -33,7 +33,10 @@ def icon(name, options = {})
       html_properties["aria-label"] = options[:aria_label]
       html_properties["role"] = options[:role]
       html_properties["aria-hidden"] = options[:aria_hidden]
-      html_properties["class"] = "icon icon--#{name} #{options[:class]}"
+
+      icon_class = "icon"
+      icon_class = "" if options[:removeIconClass]
```

Comment:
The `removeIconClass` tag comes from `decidim-design`, we will have to be careful every time we use an icon then.